frontend/packages/data-portal/app/components/Filters/IDFilter.tsx

```diff
@@ -0,0 +1,95 @@
+import { useSearchParams } from '@remix-run/react'
+import { useMemo, useState } from 'react'
+
+import { QueryParams } from 'app/constants/query'
+import { useI18n } from 'app/hooks/useI18n'
+
+import { DropdownFilterButton } from './DropdownFilterButton'
+import { InputFilter } from './InputFilter'
+
+export function IDFilter({
+  id,
+  label,
+  title,
+  queryParam,
+  prefix,
+}: {
+  id: string
+  label: string
+  title: string
+  queryParam: QueryParams
+  prefix?: string
+}) {
+  const { t } = useI18n()
+  const [searchParams, setSearchParams] = useSearchParams()
+
+  const paramValue = searchParams.get(queryParam) ?? ''
```

Comment:
since we're just using `paramValue` in a ternary, do we need the `?? ''`?

Reply:
this is done so it (and therefore `value`) will be typed as a string always instead of `string | null`